### PR TITLE
[v1.x] Make ONNX export operators work properly with the node input shape

### PR DESCRIPTION
**Pull request by josephevans**
## Description ##
Update the zeros_like and arange_like onnx export operators to properly use correct input node and shape. Updated tests for these 2 operators.

### CODE REVIEW

File: tests/python-pytest/onnx/test_operators.py [hunk 1]
```diff
@@ -91,9 +91,11 @@ def test_onnx_export_zeros_like(tmp_path):
 
 
 @pytest.mark.parametrize("dtype", ["float32", "float64"])
-def test_onnx_export_arange_like(tmp_path, dtype):
-    M = def_model('contrib.arange_like')
-    x = mx.nd.array([[-2,-1,0],[0,50,99],[4,5,6],[7,8,9]], dtype=dtype)
+@pytest.mark.parametrize("axis", [0,1,None])
+@pytest.mark.parametrize("test_data", [[[[0,1,2],[3,4,5]],[[6,7,8],[9,10,11]]], [[1,3],[2,4]]])
+def test_onnx_export_arange_like(tmp_path, dtype, axis, test_data):
```
Comment (Zha0q1) [inline tests/python-pytest/onnx/test_operators.py:96]: Can you also check the step option in the tests?

**Review by Zha0q1 — CHANGES_REQUESTED** [review]

File: tests/python-pytest/onnx/test_operators.py (hunk 1 above)
Comment (josephevans) [reply tests/python-pytest/onnx/test_operators.py]: Added.

**Review by Zha0q1 — APPROVED** [review]
LGTM!

**Review by waytrue17 — APPROVED** [review]
LGTM

**Pull request merged**